events.html

```diff
@@ -264,4 +264,21 @@ <h3 class="info-note">
 	</footer>
 </body>
 
+    <footer>
+      <h3 class="info-note">
+        To know how this page is getting developed, join our Discord Channel.
+        Contact one of the existing members
+      </h3>
+      <p class="info-repo">
+        The contents of this website are deployed from this
+        <a
+          href="https://github.com/Real-Dev-Squad/website"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          open sourced repo
```

Comment:
Why is this duplicated?

Please see the result: https://website-git-fork-prakashchoudhary07-update-event-date-change.real-dev-squad.vercel.app/events.html